src/NuGet.Core/NuGet.PackageManagement/Strings.resx

```diff
@@ -430,6 +430,7 @@
     <comment>{0} - package id</comment>
   </data>
   <data name="Warning_HttpServerUsage" xml:space="preserve">
-    <value>You are running the '{0}' operation with an 'http' source, '{1}'. Support for 'http' sources will be removed in a future version.</value>
+    <value>You are running the '{0}' operation with an 'HTTP' source, '{1}'. Non-HTTPS access will be removed in a future version. Consider migrating to an 'HTTPS' source.</value>
```

Comment:
I don't think this aligns to the new plan. You should check with @JonDouglas  on what the message should be.

Reply:
Thanks @aortiz-msft ! There are 5 Warning_HttpServerUsage in our codebase: https://github.com/search?q=repo%3ANuGet%2FNuGet.Client%20name%3D%22Warning_HttpServerUsage%22&type=code
And only one of the 5 is different. That's why I made this change to make all of them the same.
Hi @JonDouglas , could you help advise if this change is needed? Thanks!

Reply:
@aortiz-msft could you point reviewers to what plan you're referring to?

Reply:
https://github.com/NuGet/Home/pull/12542/files